commons/src/test/java/io/aiven/kafka/tieredstorage/commons/UniversalRemoteStorageManagerTest.java

```diff
@@ -77,6 +83,8 @@ class UniversalRemoteStorageManagerTest extends RsaKeyAwareTest {
     Path producerSnapshotFilePath;
     Path txnIndexFilePath;
 
+    public static final byte[] LEADER_EPOCH_INDEX_BYTES = "leader epoch index".getBytes();
+    static final int IV_SIZE = 12;
```

Comment:
`SegmentEncryptionMetadataV1` already has this constant. Maybe reuse it? We can move it later to the encryption provider where it should belong I think.